src/selectors/entities/preferences.ts

```diff
@@ -260,7 +260,6 @@ export function shouldAutocloseDMs(state: GlobalState) {
         return false;
     }
 
-    const preference = get(state, Preferences.CATEGORY_SIDEBAR_SETTINGS, Preferences.CHANNELS_SIDEBAR_AUTOCLOSE_DMS, '');
-
-    return preference.value === Preferences.AUTOCLOSE_DMS_ENABLED;
+    const preference = get(state, Preferences.CATEGORY_SIDEBAR_SETTINGS, Preferences.CHANNEL_SIDEBAR_AUTOCLOSE_DMS, '');
```

Comment:
I know this is not a new change but it is Weird that it always said DMs but infact this supports GMs as well 🤷‍♂. When ever we talk about this func it is hard to relate we do this for GMs as well.

Reply:
I think it's because the setting predates GMs if I remember correctly.